app/grants/models.py

```diff
@@ -304,6 +304,17 @@ def __str__(self):
         """Return the string representation of a Subscription."""
         return f"id: {self.pk}, active: {self.active}, subscription_hash: {self.subscription_hash}"
 
+    def successful_contribution(self, kwargs):
+        """Create a contribution object."""
+        contribution_kwargs = {
+            'tx_id': kwargs.tx_id,
+            'gas_price': kwargs.gas_price,
+            'nonce': kwargs.nonce,
+            'subscription': self
+        }
+        contribution = Contribution.objects.create(**contribution_kwargs)
+        # TODO: Email user that there payment has gone through
```

Comment:
can you generate this email in this PR?

Reply:
are there any other `TODO`s here?

Reply:
Sure, I can create the template and wire everything up. Will probably need to polish after the fact.

Reply:
I dont think there are any other todo's here